commitlint.config.js

```diff
@@ -1 +1 @@
-module.exports = {extends: ['@commitlint/config-conventional']};
+module.exports = {extends: ['@commitlint/config-angular']};
```

Comment:
as convenções do angular são até legais, mas deixa o conventional mesmo por enquanto

```suggestion
module.exports = {extends: ['@commitlint/config-conventional']};
```